test/_unit/models/file-upload.spec.js

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
Can you name this file the same as behaviour it is testing  or rename the behaviour to match the test

Reply:
Renamed to image-upload